README.md

```diff
@@ -1 +1 @@
-This is just a github test project. This doesn't meant to be any serious project. It's may be not even in a correct structure. My intention is to make it simple a possible.
+From a senior developer perspective here is nothing new to be learned. But for a youngling there is a lot to be learned from this project so don't miss any commits.
```

Comment:
I think the test is successful now you can commit to the master. Thank you for the support.

Reply:
So now I see we have to create a branch for each pull request we make to an external repo.